UPGRADE.md

```diff
@@ -200,7 +199,7 @@ The `Doctrine\DBAL\Driver::getName()` has been removed.
 
 ## BC BREAK changes the `Driver::connect()` signature
 
-The method no longer accepts the `$username`, `$password` and `$driverOptions` arguments. The corresponding values are expected to be passed as the "user", "password" and "driver_options" keys of the `$params` argument respectively.
+The method no longer accepts the `$username`, `$password` and `$driverOptions` arguments. The corresponding values are expected to be passed as the `user`, `password` and `driver_options` keys of the `$params` argument respectively.
```

Comment:
These are not symbols but strings. I'd format them as `"user"`, etc.

Reply:
IMO array keys still deserve that formatting, especially since `driver_options` contains an underscore, but also because the user will get an issue if they misspell these strings.

Reply:
I meant both formatting as code and quotes.

Reply:
Oh ok, let's do this!

Reply:
I spent 5 minutes before answering googling how to represent a backtick as code in markdown and gave up, although I remember I found a solution earlier.

Reply:
You can edit my message to see how

``` `"hi"` ```

Reply:
How fun is it? This is the article I had found earlier!